core/domain/classifier_services.py

```diff
@@ -460,15 +455,14 @@ def create_classifier_training_job_for_reverted_exploration(
     state_names = exploration_to_revert_to.states.keys()
     for index, classifier_training_job in enumerate(
             classifier_training_jobs_for_old_version):
-        if classifier_training_job is None:
-            continue
```

Comment:
I like this more because it uses less nesting — any technical reason you changed it?

Reply:
I changed it as I couldn't test the continue statement. But the new changes work the same, don't they? The condition only works if `classifier_training_job` is not None

Reply:
OK, then please add an explicit check:

```python
if classifier_training_job is not None:
    # ...
```

Reply:
done